src/Commands/AdminCommands/CleanupCommand.php

```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```

Comment:
Very rare case would be an admin who didnt message his bot in private chat yet.
All admins commands should get some kind of change like this, maybe we could decide on something and do it in this PR?